packages/sdks/snippets/hydrogen/app/components/NavBar.tsx

```diff
@@ -0,0 +1,16 @@
+import {type BuilderContent} from '@builder.io/sdk-react';
+
+export function NavBar({links}: {links: BuilderContent | null}) {
```

Comment:
you're typing links as `BuilderContent | null` but it will only get passed to this component when links is defined because of `{links && <NavBar links={links} />}`